fe-nc-games/src/components/CommentsList.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useParams } from 'react-router-dom';
+import CommentCard from './CommentCard';
+import getComments from '../utils/apiRequests';
+
+function CommentsList() {
+  const [commentsList, setCommentsList] = useState([]);
+  const [isLoading, setLoading] = useState(true);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    getComments(review_id)
+    .then((response) => {
+      setCommentsList(response.data);
+      setLoading(false);
+    });
+  }, []);
+
+  if (isLoading) {
+    return <p>Loading Items...</p>;
+  }
+
+  return (
+    <>
+      <section>
```

Comment:
When there are no comments, please indicate that to the user with a message like: 'No comments yet'.

Reply:
Added if statement:
  if (commentsList.length === 0) {
    return <p>No comments yet, why not add one?</p>
  }